build/css/Forum.css

```diff
@@ -0,0 +1,36 @@
+.input[type=text] {
+    width: 130px;
+    -webkit-transition: width 0.4s ease-in-out;
+    transition: width 0.4s ease-in-out;
+}
+
+/* When the input field gets focus, change its width to 100% */
+input[type=text]:focus {
```

Comment:
I would be careful with css like this, it applies globally to all elements referencing that stylesheet.